aten/src/ATen/core/ivalue.cpp

```diff
@@ -763,11 +763,6 @@ IValueComparator getGreaterThanComparator(const IValue& v) {
   };
 }
 
-static std::ostream& operator<<(std::ostream& out, const ivalue::EnumHolder& v) {
```

Comment:
This is very nice to have around when doing debugging. Can we keep this please?

Reply:
Recover it